.eslintrc

```diff
@@ -36,8 +36,10 @@
     "describe": true,
     "fdescribe": true,
     "xdescribe": true,
+    "context": true,
```

Comment:
I'm :-1: on changing our testing language (at least in this PR).

* It's strictly a wording change, not a functional one, so it doesn't support this PR's goal of improving the stability of the GitHub package suite.
* In this context, I feel more strongly about _consistency_ than I do about the "correctness" of either option.
  * We have a giant test suite and it would take a long time to percolate this change through it incrementally. So, introducing this change the more natural way would result in an extended period of inconsistency.
  * Doing it all at once in a batch change has similar drawbacks to linter changes: it introduces spurious conflicts with open PRs, it interferes with blame data, and it's a hard diff to validate visually since it's so repetitive.
* There are a lot of other systematic things I'd like to fix throughout our test suite before we addressed this. For example, we currently have about three distinct ways to create fixture data.
* I have some [`init.js` Atom hacks](https://gist.github.com/smashwilson/79b018bc7c9c1732a698b0d2a188986f) built around intelligently folding `describe` / `it` suites that I've shared around.
* Honestly, I'm more accustomed to `describe` and `it` at this point. I think they read reasonably well.

Reply:
reverted